src/main/java/cubicchunks/worldgen/dependency/Requirement.java

```diff
@@ -24,27 +24,19 @@
 
 package cubicchunks.worldgen.dependency;
 
-import cubicchunks.util.AddressTools;
+import cubicchunks.util.CubeCoords;
 import cubicchunks.worldgen.GeneratorStage;
 
 public class Requirement {
 
-	public int cubeX;
-	public int cubeY;
-	public int cubeZ;
+	public CubeCoords coords;
 	public GeneratorStage targetStage;
```

Comment:
Change these to private and make getter methods for them (I didn't notice it before). Generally public fields are not good (at least in java).